components/infobox/commons/infobox_team.lua

```diff
@@ -168,9 +169,17 @@ function Team:_createLocation(location)
 		return ''
 	end
 
+	local locationDisplay = self:getStandardLocationValue(location)
+	if String.isNotEmpty(locationDisplay) then
+		locationDisplay = '[[:Category:' .. locationDisplay
+			.. '|' .. locationDisplay .. ']]'
+	end
+	local demonym = Localisation.getLocalisation(locationDisplay)
```

Comment:
Aren't you giving Module:Localisation a string in the category format here?

Reply:
yeah, should be fixed now